### DISCUSSION TITLE
Ambassador Quiz

### DISCUSSION THREAD

**Discussion by abdullahmorrison** [alveusgg/extension#117] [Ideas]
ambassador quiz that popups up 1 random question for the chat to answer to track how well education is sticking. This could happen during stream f's or slower parts of a stream.

Stats of the quiz could be displayed in the config page of the extension.

A full quiz would be better on the website. Having to go to a new site to do things is a lot of friction so if the quiz can be added to the extension, it would be best.

**Comment by MattIPv4** [comment]
cc https://github.com/alveusgg/alveusgg/issues/291

What'd be really cool is having full quizzes defined on the site, and then an extension command that triggers a popup with a random question (or a question from a given category etc.) pulled from the site. That'd save us from duplicating data and whatnot and would tie the two together nicely.
